src/cliAction/login/handler.go

```diff
@@ -140,16 +140,16 @@ func (h *Handler) loginWithPassword(_ context.Context, login, password string) e
 
 	data := h.storage.Data()
 	data.Token = tokenResponseObject.Token
```

Comment:
I might miss something but do we need the same code here and on line 144? 🤔

Reply:
Yes, we don't.